### PR TITLE
fix: symfony 6 compatibility

### PR DESCRIPTION
**Pull request by tuxes3**
Hey.

In [this PR](https://github.com/jorisdugue/h5p-bundle/pull/44) I fixed the symfony 6 compatibility, but actually just checked weather it compiles after `bin/console c:c`. 

With this PR I made it work with the method changes/deprecations etc.

Furthermore I added `phpstan` for static code analyses.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,16 +32,20 @@
...
+    "symfony/asset": "~5.0|~6.0|~7.0",
+    "symfony/form": "~5.0|~6.0|~7.0"
   },
   "autoload": {
     "psr-4": {
       "Studit\\H5PBundle\\": ""
     }
+  },
+  "require-dev": {
+    "phpstan/phpstan": "^1.10"
```
Comment (jorisdugue) [inline composer.json:49]: Maybe using a ci for better no ?

File: Core/H5PSymfony.php [hunk 2]
```diff
@@ -1029,6 +1044,6 @@ private function truncateTable($tableClassName)
      */
     public function libraryHasUpgrade($library)
```
Comment (jorisdugue) [inline Core/H5PSymfony.php:1045]: maybe implement the typing too ? 
```suggestion
    public function libraryHasUpgrade($library): bool
```

File: Controller/H5PInteractionController.php [hunk 3]
```diff
@@ -153,43 +174,13 @@ public function embedAction(Request $request, Content $content)
                 ],
             ],
         ];
-        $h5p_content = $content;
```
Comment (jorisdugue) [inline Controller/H5PInteractionController.php:156]: I'm not sure whether this part of the code should be deleted, can you tell me why?

**Review by jorisdugue — CHANGES_REQUESTED** [review]

File: Controller/H5PInteractionController.php (hunk 3 above)
Comment (tuxes3) [reply Controller/H5PInteractionController.php:156]: Yes you are right, i deleted the wrong side of the `if`. Thanks for checking!

File: composer.json (hunk 1 above)
Comment (tuxes3) [reply composer.json:49]: i created a workflow

**Review by jorisdugue — APPROVED** [review]

**Comment by jorisdugue** [comment]
Thank you for the different modifications that should be good for now :)

**Comment by tuxes3** [comment]
could you give a tag please

**Comment by jorisdugue** [comment]
> could you give a tag please

Hey i forgot :( 

this is now done 😄 https://github.com/jorisdugue/h5p-bundle/releases/tag/v2.0.1

**Pull request merged**